magit-stgit.el

```diff
@@ -593,66 +593,70 @@ ask for confirmation before deleting."
 
 ;;;; magit-stgit-mail
 
-(magit-define-popup magit-stgit-mail-popup
-  "Popup console for StGit mail."
-  'magit-stgit-commands
+(transient-define-prefix magit-stgit-mail ()
```

Comment:
One of the few breaking changes in StGit 2.0 is that the `stg mail` command is replaced by `stg format-patch` and `stg send-email`, which are thin wrappers on `git format-patch` and `git send-email`, respectively.

I am going to be releasing StGit 2.0 shortly (days, not weeks). My preference is for this extension to target StGit 2.0 and not also try to support StGit 1.x.

For purpose of this initial set of changes, converting the existing support for `stg mail` to transient makes sense, so I'm not asking for any changes here, but moving forward I think this code should be dropped or morphed to support `stg send-email`.

Reply:
Ok, makes sense. Let's leave it as it is, and then we can morph it into what's appropriate for StGit 2.0 soon.